packages/web/src/lib/web-mode.ts

```diff
@@ -234,6 +234,21 @@ export class WebMode {
 		}
 	}
 
+	/**
+	 * @description Sign a Logic Signature transaction
+	 * @param transaction algosdk.Transaction object
+	 * @param logicSig Logic Sig Account
+	 * @returns Returns txID and blob object
+	 */
+	signLogicSignatureTxn(transaction: Transaction, logicSig: algosdk.LogicSigAccount): { txID: string, blob: Uint8Array } {
+		try {
+			return algosdk.signLogicSigTransaction(transaction, logicSig)
+		} catch (err) {
+			error(err);
+			throw new Error("Error while signing Lsig Transaction" + err);
+		}
+	}
```

Comment:
this code is copy pasted. Let's move it to a separate function, and here let's just use one liner.

Reply:
okay